CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# Unreleased
+
+* Add the `derive` feature that enables all derive-related smaller features
+  (`specs-derive` and `shred-derive` currently). ([#687])
+
+[#687]: https://github.com/amethyst/specs/pull/687
```

Comment:
Should I be adding (and guessing the PR number) manually, or is there an automatic way how this happens?

Also, the other links seem to be to slide-rs/specs, but this repo is amethyst/specs. Is that a previous place where it lived?

Reply:
Heya, it's manual for now. Normally I would look at the most recent issues/PR and add one (ya, not a great method).

Yeap it used to live in `slide-rs` before moving into `amethyst` (shall update the links in a later PR).